quartermaster/src/main/java/com/github/reflectoring/infiniboard/quartermaster/widget/rest/WidgetConfigResourceAssembler.java

```diff
@@ -0,0 +1,48 @@
+package com.github.reflectoring.infiniboard.quartermaster.widget.rest;
+
+import java.sql.Date;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+
+import org.springframework.hateoas.mvc.ResourceAssemblerSupport;
+
+import com.github.reflectoring.infiniboard.packrat.source.SourceConfig;
+import com.github.reflectoring.infiniboard.packrat.widget.WidgetConfig;
+import com.github.reflectoring.infiniboard.quartermaster.dashboard.rest.DashboardController;
+
+import static org.springframework.hateoas.mvc.ControllerLinkBuilder.linkTo;
+import static org.springframework.hateoas.mvc.ControllerLinkBuilder.methodOn;
+
+public class WidgetConfigResourceAssembler extends ResourceAssemblerSupport<WidgetConfig, WidgetConfigResource> {
+
+    private Integer dashboardId;
+
+    public WidgetConfigResourceAssembler(Integer dashboardId) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUndocumentedApi)